starters/cra-rxjs-styled-components/src/components/Loader/Loader.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Loader } from './Loader';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Add Loader', () => {
+  it('should appear while waiting for response', () => {
+    const { container } = render(<Loader />, { wrapper: MemoryRouter });
+    expect(container.getElementsByClassName('w-60').length).toBe(1);
```

Comment:
It's not recommended to select elements from the app by the Tailwind classes because they are not unique at all, you can replace it with `data-testid`